addons/payment_test/tests/__init__.py

```diff
@@ -0,0 +1,6 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from . import common
+from . import test_payment_test
+from . import test_token_payment_test
+from . import test_auth_and_capture_payment_test
```

Comment:
I can't have one file because the `setUpClass()` needed is different for each file. I'm not too fond of those names either but I wanted to follow other acquirers' nomenclature.

Here is my suggestion to improve the name of the files:
```suggestion
from . import test_acquirer
from . import test_token
from . import test_manual_capture
```

Reply:
Way better already :+1:
Maybe simply `test_capture`, it tests the capture flows

Reply:
Eventually, I changed my mind. I gathered all tests in `test_acquirer.py`